pkg/pxc/app/statefulset/haproxy.go

```diff
@@ -450,6 +450,16 @@ func (c *HAProxy) Volumes(podSpec *api.PodSpec, cr *api.PerconaXtraDBCluster, vg
 			},
 		)
 	}
+	if cr.CompareVersionWith("1.15.0") >= 0 {
+		vol.Volumes = append(vol.Volumes,
+			corev1.Volume{
+				Name: app.EtcVolumeName,
+				VolumeSource: corev1.VolumeSource{
+					EmptyDir: &corev1.EmptyDirVolumeSource{},
+				},
+			},
+		)
+	}
```

Comment:
is this necessary?

Reply:
[f34997c](https://github.com/percona/percona-xtradb-cluster-operator/pull/1753/commits/f34997c006081c906fd3b085f949e176fa436912)